### PR TITLE
Add TKBC2 article

### PR DESCRIPTION
**Pull request by Hoshikuz**
The process is all done.

### CODE REVIEW

File: wiki/Contests/TKBC/2/en.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | Profile Badge |
+| ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | 2 months of supporter |
+| ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of supporter |
+
+## Organisation
+
+The Taiko Korean Beatmap Contest #2 is run by members of the Korean osu!taiko mapping community.
```
Comment (Nivalyx) [inline wiki/Contests/TKBC/2/en.md:59]: ```The Taiko Korean Beatmap Contest #2 was run by members of the Korean osu!taiko mapping community.```

File: wiki/Contests/TKBC/2/en.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+| Position | Member(s) |
+| :-- | :-- |
+| Manager | [[Karcher] 2](https://osu.ppy.sh/users/9892196), [Civil oath](https://osu.ppy.sh/users/3216107) |
```
Comment (Nivalyx) [inline wiki/Contests/TKBC/2/en.md:63]: a lot of [escape characters](http://bookdown.org/yihui/rmarkdown-cookbook/special-chars.html) are seeming to be missing on these tables at the moment, namely :

- `[[Karcher] 2]` should be --> `[\[Karcher\] 2]`
- `[4_4]` should be --> `[4\_4]`

**Review by Nivalyx — COMMENTED** [review]
the whole pr seems to be alright but i would like to specifically point out that right now you're missing the flags which are supposed to be attached to the corresponding staffs/winners username as follows :

image

``![][flag_KR]``
``[flag_KR]: /wiki/shared/flag/KR.gif "South Korea"``

File: wiki/Contests/TKBC/2/en.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+
+![TKBC2 Logo](img/logo.png)
+
+The **Taiko Korean Beatmap Contest #2** (***TKBC2***) was a osu!taiko mapping contest for Koreans. It was the second installment of the TKBC.
+
+## Contest schedule
+
+| Event | Timestamp (UTC +9) |
+| --: | :-- |
+| Submitting Phase | 2020-11-30/2020-12-27 |
```
Comment (TPGPL) [inline wiki/Contests/TKBC/2/en.md:17]:
```suggestion
| Submission phase | 2020-11-30/2020-12-27 |
```

File: wiki/Contests/TKBC/2/en.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+| Judging Phase | 2020-12-28/2021-01-17 |
+| Results Announcement | 2021-01-17 |
```
Comment (TPGPL) [inline wiki/Contests/TKBC/2/en.md:19]:
```suggestion
| Judging phase | 2020-12-28/2021-01-17 |
| Results announcement | 2021-01-17 |
```

File: wiki/Contests/TKBC/2/en.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+- Only Koreans can participate this contest.
+- Judges can't participate this contest.
+
+### Metadata rules
+
+- Artist must be `asdf`.
+- Title must be `1223`.
+- Difficulty must be Oni or Inner Oni.
+- Source and tags must be left blank.
+- When submitting your map, include the metadata in the email.
```
Comment (TPGPL) [inline wiki/Contests/TKBC/2/en.md:42]:
```suggestion
- While submitting your beatmap, include the metadata in the email.
```

File: wiki/Contests/TKBC/2/en.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+- This contest is for osu!taiko mode.
+- You should choose a song which matches the theme.
+- The drain time should be not less than 80 and not more than 150.
+- The difficulty should be Oni or higher.
+- You must follow the [Ranking Criteria](/wiki/Ranking_Criteria).
+- If you submit several maps, the last one would be your entry.
+- You should not reveal your map before the results announcing.
+- You can't submit what you uploaded already.
+- Only Koreans can participate this contest.
```
Comment (TPGPL) [inline wiki/Contests/TKBC/2/en.md:33]:
```suggestion
- Only users from South Korea can participate in this contest.
```

File: wiki/Contests/TKBC/2/en.md [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+### Contest rules
+
+- This contest is for osu!taiko mode.
+- You should choose a song which matches the theme.
+- The drain time should be not less than 80 and not more than 150.
+- The difficulty should be Oni or higher.
+- You must follow the [Ranking Criteria](/wiki/Ranking_Criteria).
+- If you submit several maps, the last one would be your entry.
+- You should not reveal your map before the results announcing.
+- You can't submit what you uploaded already.
```
Comment (TPGPL) [inline wiki/Contests/TKBC/2/en.md:32]:
```suggestion
- You can't submit a beatmap you have already uploaded.
```

File: wiki/Contests/TKBC/2/en.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+
+### Contest rules
+
+- This contest is for osu!taiko mode.
+- You should choose a song which matches the theme.
+- The drain time should be not less than 80 and not more than 150.
+- The difficulty should be Oni or higher.
+- You must follow the [Ranking Criteria](/wiki/Ranking_Criteria).
+- If you submit several maps, the last one would be your entry.
+- You should not reveal your map before the results announcing.
```
Comment (TPGPL) [inline wiki/Contests/TKBC/2/en.md:31]:
```suggestion
- You should not reveal your map before the results announcement.
```

File: wiki/Contests/TKBC/2/en.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+## Ruleset
+
+### Contest rules
+
+- This contest is for osu!taiko mode.
+- You should choose a song which matches the theme.
+- The drain time should be not less than 80 and not more than 150.
+- The difficulty should be Oni or higher.
+- You must follow the [Ranking Criteria](/wiki/Ranking_Criteria).
+- If you submit several maps, the last one would be your entry.
```
Comment (TPGPL) [inline wiki/Contests/TKBC/2/en.md:30]:
```suggestion
- If you submit several beatmaps, the last one will be your entry.
```

File: wiki/Contests/TKBC/2/en.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+| Judging Phase | 2020-12-28/2021-01-17 |
+| Results Announcement | 2021-01-17 |
+
+## Ruleset
+
+### Contest rules
+
+- This contest is for osu!taiko mode.
+- You should choose a song which matches the theme.
+- The drain time should be not less than 80 and not more than 150.
```
Comment (TPGPL) [inline wiki/Contests/TKBC/2/en.md:27]:
```suggestion
- The drain time should not be less than 80 and more than 150.
```

File: wiki/Contests/TKBC/2/en.md [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+| --: | :-- |
+| Submitting Phase | 2020-11-30/2020-12-27 |
+| Judging Phase | 2020-12-28/2021-01-17 |
+| Results Announcement | 2021-01-17 |
+
+## Ruleset
+
+### Contest rules
+
+- This contest is for osu!taiko mode.
```
Comment (TPGPL) [inline wiki/Contests/TKBC/2/en.md:25]:
```suggestion
- This contest is for osu!taiko mode.
- The theme for this installment is "Snow" or "Star".
```

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Comment by TPGPL** [comment]
English article looks fine now, you should apply the changes to the Korean translation

**Comment by TPGPL** [comment]
@Hoshikuz could you apply the changes to the Korean translation as well, please?

**Comment by Hoshikuz** [comment]
I forgot this
sorry :c

File: wiki/Contests/TKBC/2/ko.md [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+
+### 메타데이터 규칙
+
+- Artist는 asdf를 사용해주시기 바랍니다.
+- Title은 1223을 사용해주시기 바랍니다.
+- Difficulty는 Oni 혹은 Inner Oni를 사용해주시기 바랍니다.
+- Source랑 Tags는 전부 비워주세요.
+- 이메일에 아티스트와 곡명, 그리고 주제와 연관되는 간략한 이유를 포함해서 제출해주시기 바랍니다.
+
+[flag_KR]: /wiki/shared/flag/KR.gif "South Korea"
```
Comment (TPGPL) [inline wiki/Contests/TKBC/2/ko.md:75]:
```suggestion
[flag_KR]: /wiki/shared/flag/KR.gif "대한민국"
```

File: wiki/Contests/TKBC/2/ko.md [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+
+- [포럼 스레드](https://osu.ppy.sh/community/forums/topics/1162734)
+
+## 결과
+
+| 순위 | 매퍼 |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ![][flag_KR] [applerss](https://osu.ppy.sh/users/983349) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ![][flag_KR] [4\_4](https://osu.ppy.sh/users/1152851) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ![][flag_KR] [ddm](https://osu.ppy.sh/users/7910282) |
```
Comment (TPGPL) [inline wiki/Contests/TKBC/2/ko.md:49]: the alt texts and tooltips of images have to be translated

for reference:

`alt-text`

File: wiki/Contests/TKBC/2/ko.md [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+| Designer | ![][flag_KR] [oL0](https://osu.ppy.sh/users/1134683) |
+| Judge | ![][flag_KR] [\[Karcher\] 2](https://osu.ppy.sh/users/9892196), ![][flag_KR] [Petit](https://osu.ppy.sh/users/4637369), ![][flag_KR] [Ozu](https://osu.ppy.sh/users/980092) |
+
+## 상품
+
+| 순위 | 상품 |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 프로필 뱃지 |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 서포터 2개월 |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 서포터 1개월 |
```
Comment (TPGPL) [inline wiki/Contests/TKBC/2/ko.md:37]: the alt texts and tooltips of images have to be translated

File: wiki/Contests/TKBC/2/ko.md [hunk 15]
```diff
@@ -0,0 +1,75 @@
+---
+tags:
+- TKBC
+- TKBC2
+---
+
+# Taiko Korean Beatmap Contest #2
+
+![TKBC2 logo](img/logo.png)
```
Comment (TPGPL) [inline wiki/Contests/TKBC/2/ko.md:9]: `TKBC2 logo` has to be translated

File: wiki/Contests/TKBC/2/ko.md [hunk 16]
```diff
@@ -0,0 +1,75 @@
...
+
+## 주최
+
+Taiko Korean Beatmap Contest #2는 한국 태고 매핑 커뮤니티 인원들에 의해 주최됩니다.
+
+| 직책 | 인원 |
+| :-- | :-- |
+| Manager | ![][flag_KR] [\[Karcher\] 2](https://osu.ppy.sh/users/9892196), ![][flag_KR] [Civil oath](https://osu.ppy.sh/users/3216107) |
+| Designer | ![][flag_KR] [oL0](https://osu.ppy.sh/users/1134683) |
+| Judge | ![][flag_KR] [\[Karcher\] 2](https://osu.ppy.sh/users/9892196), ![][flag_KR] [Petit](https://osu.ppy.sh/users/4637369), ![][flag_KR] [Ozu](https://osu.ppy.sh/users/980092) |
```
Comment (TPGPL) [inline wiki/Contests/TKBC/2/ko.md:29]: i think the position names can be translated as well

**Review by TPGPL — CHANGES_REQUESTED** [review]

File: wiki/Contests/TKBC/2/ko.md [hunk 17]
```diff
@@ -0,0 +1,75 @@
...
+
+- [포럼 스레드](https://osu.ppy.sh/community/forums/topics/1162734)
+
+## 결과
+
+| 순위 | 매퍼 |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1위") | ![][flag_KR] [applerss](https://osu.ppy.sh/users/983349) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2위") | ![][flag_KR] [4\_4](https://osu.ppy.sh/users/1152851) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3위") | ![][flag_KR] [ddm](https://osu.ppy.sh/users/7910282) |
```
Comment (TPGPL) [inline wiki/Contests/TKBC/2/ko.md:49]: `Gold crown`, `Silver crown`, `Bronze crown` needs to be translated

File: wiki/Contests/TKBC/2/ko.md [hunk 18]
```diff
@@ -0,0 +1,75 @@
...
+| 디자인 | ![][flag_KR] [oL0](https://osu.ppy.sh/users/1134683) |
+| 심사 | ![][flag_KR] [\[Karcher\] 2](https://osu.ppy.sh/users/9892196), ![][flag_KR] [Petit](https://osu.ppy.sh/users/4637369), ![][flag_KR] [Ozu](https://osu.ppy.sh/users/980092) |
+
+## 상품
+
+| 순위 | 상품 |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1위") | 프로필 뱃지 |
+| ![Silver crown](/wiki/shared/crown-silver.png "2위") | 서포터 2개월 |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3위") | 서포터 1개월 |
```
Comment (TPGPL) [inline wiki/Contests/TKBC/2/ko.md:37]: `Gold crown`, `Silver crown`, `Bronze crown` needs to be translated

**Review by TPGPL — CHANGES_REQUESTED** [review]

**Review by TPGPL — APPROVED** [review]

**Pull request merged**